compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEConvertToDma.cpp

```diff
@@ -387,32 +403,35 @@ void AMDAIEConvertToDmaPass::runOnOperation() {
   // step. This is easy to implement, but not the most direct lowering, so
   // we might want to revisit this.
   WalkResult convertCopiesWalkResult =
-      getOperation()->walk([&rewriter](linalg::CopyOp copyOp) {
+      getOperation()->walk([&](linalg::CopyOp copyOp) {
```

Comment:
Why?

Reply:
I don't think there's need to pass `&rewriter`. And I'm trying to follow the coding styles in llvm-project which just pass [&] in such walking functions, e.g. https://github.com/llvm/llvm-project/blob/e1e788f423b5c780c40912ab102b0a3c4b92b9de/mlir/lib/Dialect/SCF/Transforms/ForallToFor.cpp#L63